lib/open_food_network/paperclippable.rb

```diff
@@ -4,18 +4,18 @@
 module OpenFoodNetwork
   module Paperclippable
     def self.included(base)
-      base.send :extend, ActiveModel::Naming
-      base.send :extend, ActiveModel::Callbacks
-      base.send :include, ActiveModel::Validations
-      base.send :include, Paperclip::Glue
+      base.extend(ActiveModel::Naming)
+      base.extend(ActiveModel::Callbacks)
+      base.include(ActiveModel::Validations)
+      base.include(Paperclip::Glue)
 
       # Paperclip required callbacks
-      base.send :define_model_callbacks, :save, only: [:after]
-      base.send :define_model_callbacks, :commit, only: [:after]
-      base.send :define_model_callbacks, :destroy, only: [:before, :after]
+      base.define_model_callbacks(:save, only: [:after])
```

Comment:
it is public https://apidock.com/rails/ActiveModel/Callbacks/define_model_callbacks